src/main/java/core/basesyntax/FileWork.java

```diff
@@ -8,7 +13,31 @@
  * Результат: web wide width world</p>
  */
 public class FileWork {
+    public static final String W = "w";
+
     public String[] readFromFile(String fileName) {
-        return null;
+        ArrayList<String> wordsArray = new ArrayList<>();
+        StringBuilder stringBuilder = new StringBuilder(" ");
+        String line;
+        try (FileReader fileReader = new FileReader(fileName)) {
+            BufferedReader bufferedReader = new BufferedReader(fileReader);
```

Comment:
bufferedReader should be closed after use. so we need add this to try with resources like FileReader

Reply:
done